src/migration/1695912851708-migration.ts

```diff
@@ -0,0 +1,112 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Migration1695912851708 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const schemaName = process.env.NODE_ENV === 'test' ? 'test' : 'public';
+        await queryRunner.query(`
+            DROP TABLE IF EXISTS ${schemaName}.etl_log
+        `);
+        await queryRunner.query(`
+            DROP TYPE IF EXISTS job_status_type
+        `);
+        await queryRunner.query(`
+            CREATE TYPE job_status_type AS ENUM ('Pass', 'Fail', 'In Progress');
+        `);
+        await queryRunner.query(`
+            CREATE TABLE ${schemaName}.etl_log (
```

Comment:
@anayanapalli Let me know if any other fields are needed or if the names should be changed